lib/awesome_nested_set/model.rb

```diff
@@ -173,7 +173,7 @@ def without_self(scope)
         end
 
         def store_new_parent
-          @move_to_new_parent_id = send("#{parent_column_name}_changed?") ? parent_id : false
+          @move_to_new_parent_id = send("saved_change_to_#{parent_column_name}?") ? parent_id : false
```

Comment:
This one gets called from a `before_save` hook.  Should be `will_save_change_to`.

https://github.com/Kindful/awesome_nested_set/blob/master/lib/awesome_nested_set/awesome_nested_set.rb#L68

Reply:
Done.